x-pack/solutions/security/plugins/security_solution/public/management/components/artifact_entry_card/artifact_entry_card_minified.tsx

```diff
@@ -29,12 +29,12 @@ import type { ArtifactEntryCardDecoratorProps } from './artifact_entry_card';
 
 const CardContainerPanel = styled(EuiSplitPanel.Outer)`
   &.artifactEntryCardMinified + &.artifactEntryCardMinified {
-    margin-top: ${({ theme }) => theme.eui.euiSizeL};
+    margin-top: ${({ theme }) => theme.euiTheme.size.l};
   }
 `;
 
 const CustomSplitInnerPanel = styled(EuiSplitPanel.Inner)`
-  background-color: ${({ theme }) => theme.eui.euiColorLightestShade} !important;
+  background-color: ${({ theme }) => theme.euiTheme.colors.lightestShade} !important;
```

Comment:
looks like `lightestShade` is deprecated - since this PR already touches this line, do you think we can use another color here?